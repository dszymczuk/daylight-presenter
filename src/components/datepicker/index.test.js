import React from "react";
import { render, cleanup } from '@testing-library/react';
import DatePicker from "./index";

afterEach(cleanup);

it("renders DatePicker", () => {
  const props = {
    field: {
      name: "datePicker",
      value: new Date()
    },
    form: {
      setFieldValue: () => {
      },
      setFieldTouched: () => {
      }
    }
  };

  const { asFragment } = render(<DatePicker {...props}/>);
  expect(asFragment()).toMatchSnapshot();
});