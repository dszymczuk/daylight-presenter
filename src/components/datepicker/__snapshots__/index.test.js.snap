// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DatePicker 1`] = `
<DocumentFragment>
  <div
    class="datepickerWrapper"
  >
    <div
      class="form-group"
    >
      <label
        class="form-label"
        for="datePicker"
      >
        Date
      </label>
      <div
        class="react-datepicker-wrapper"
      >
        <div
          class="react-datepicker__input-container"
        >
          <input
            class="form-control"
            id="datePicker"
            value="26.01.2020"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
